src/unexploitable_search/tasks/alpaca/scorer.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any
+
+from inspect_ai.scorer import (
+    Score,
+    Scorer,
+    Target,
+    accuracy,
+    model_graded_qa,
+    scorer,
+)
+from inspect_ai.solver import TaskState
+
+from unexploitable_search.tasks.alpaca.scorer_templates import ALPACA_SCORER_TEMPLATE
+from unexploitable_search.tasks.alpaca.side_quest_prompts import (
+    SIDE_QUEST_SCORER_PROMPTS,
+)
+from unexploitable_search.tasks.alpaca.scorer_templates import (
+    SIDE_QUEST_SCORER_TEMPLATE,
+)
+
+
+@scorer(metrics=[accuracy()])
+def alpaca_correctness() -> Scorer:
+    return model_graded_qa(
+        model="openai/gpt-4o-mini",
+        template=ALPACA_SCORER_TEMPLATE,
+    )
+
+
+@scorer(metrics=[accuracy()])
+def side_quest_correctness(side_quest: str, side_quest_args: dict[str, Any]) -> Scorer:
+    if side_quest == "word":
```

Comment:
Future work. It might be worth to compile all the side quest execution details on a single place, to make it easy to add or verify the details of the sidequest. Check out how we're doing it on the coding tasks. We could harmonize this for both implementations